packages/experiments-realm/crm/account.gts

```diff
@@ -127,6 +131,166 @@ class IsolatedTemplate extends Component<typeof Account> {
     );
   }
 
+  get realmURL(): URL {
+    return this.args.model[realmURL]!;
+  }
+
+  get realmHrefs() {
+    return [this.realmURL?.href];
+  }
+
+  // Query All Active Deal that linked to current Account
+  get dealQuery(): Query {
+    return {
+      filter: {
+        on: {
+          module: new URL('./deal', import.meta.url).href,
+          name: 'Deal',
+        },
+        every: [
+          { eq: { 'account.id': this.args.model.id ?? '' } },
+          {
+            not: {
+              any: [
+                { eq: { 'status.label': 'Closed Won' } },
+                { eq: { 'status.label': 'Closed Lost' } },
```

Comment:
I thought there is a computed `isActive`??

Reply:
Maybe you have to add `isUsed` inside the `isActive` field bcos we don't use `isActive` in the isolated template

Reply:
You need to add `isUsed` otherwise the query will return no data